src/default_panels/GraphCreatePanel.js

```diff
@@ -66,6 +68,38 @@ const GraphCreatePanel = ({localize: _}) => {
         <DataSelector label={_('Columns')} attr="cells.values" />
       </Section>
 
+      <TraceTypeSection
+        name={_('Data')}
+        traceTypes={['scatterpolar', 'scatterpolargl']}
+      >
+        <DataSelector label={_('Radius')} attr="r" />
+        <DataSelector label={_('Theta')} attr="theta" />
+        <Dropdown
+          label={_('Theta Unit')}
+          options={[
+            {label: _('Radians'), value: 'radians'},
+            {label: _('Degrees'), value: 'degrees'},
+            {label: _('Gradians'), value: 'gradians'},
+          ]}
+          attr="thetaunit"
+          clearable={false}
+        />
+        <LayoutNumericSlider
+          attr="polar.sector[0]"
+          label={_('Theta Start')}
+          min={0}
+          max={360}
+          showSlider
```

Comment:
Do we need a new `LayoutNumericSlider` if we're going to use `showSlider`? Can't we just use `Numeric` with `showSlider` ?

Reply:
Or a more general `LayoutNumeric` or something?